### DISCUSSION TITLE
"Windows protected your PC" alert with self built client

### DISCUSSION THREAD

**Discussion by computman007** [rustdesk/rustdesk#5897] [Q&A]
Hello

My users get a warning message on Windows when using a self built version of the client using the Github actions

Using the official release does not raise any alerts but need to manually add the servers settings for each user

What to update on github actions to avoid this alert ?

I have followed those steps https://rustdesk.com/docs/en/dev/build/all/

![image](https://github.com/rustdesk/rustdesk/assets/25824896/f3c3e491-9b79-4b8a-b3aa-842dd43a3c77)

**Comment by discoveryX64** [comment]
Normally windows does that because the program isn't signed. You need to pay like 200 something a year or month to sign it. You can just run it anyways and nothing should happen. If you really want to check if it isn't, use VirusTotal or something else, or run it on a vm.

**Reply by discoveryX64** [reply]
I think you can download the file (the certificate) to sign it under them? Not sure if that's true though

**Comment by SergeyMy** [comment]
![изображение](https://github.com/rustdesk/rustdesk/assets/131688106/c67c574e-ed1a-425c-8f85-b073518ac9a0)

![изображение](https://github.com/rustdesk/rustdesk/assets/131688106/232b3cce-cafa-4000-8689-5ed95c208b04)

these variables must be set in “Settings -> Secrets and variables -> Actions”.

**Reply by computman007** [reply]
I have to copy and paste this somewhere ?

I must admit I don't know how to do it

**Reply by SergeyMy** [reply]
This is a qualified electronic signature, and in order not to swear at it, it must be confirmed in one of the CS.
as stated above - it usually costs money, but does not make any sense! :)

this means: some Jobs confirms that the code was signed by @computman007, and he also pays this Jobs for it :):):):)
